lib/offline_geocoder.rb

```diff
@@ -0,0 +1,46 @@
+require "offline_geocoder/version"
+require "csv"
+require "kdtree"
+
+class OfflineGeocoder
+  def initialize
+    unless defined? @@cities
```

Comment:
I understand that this stores the values on class variables to prevent to re initialize everything again it when you create a new instance. But (i'm not sure about it, so i ask) will this be always on memory? will it be recycled be the garbage collector?
And at the same time i thing no one will create many instances of this. I don't know. What do you thing @sildur and @elbrujohalcon about this?